source/common/router/config_impl.h

```diff
@@ -344,6 +344,8 @@ class VirtualHostImpl : public VirtualHost, Logger::Loggable<Logger::Id::router>
 };
 
 using VirtualHostSharedPtr = std::shared_ptr<VirtualHostImpl>;
+// Sentinel value to represent an unset std::duration.
+static constexpr long UnsetDuration = -1;
```

Comment:
looking at the definition in one of our header files, and also to align with Envoy practices, I think this should be int64_t rather than long.

Reply:
Done